GVRf/Framework/jni/objects/textures/texture.h

```diff
@@ -41,6 +41,11 @@ class Texture: public RecyclableObject {
     }
 
     virtual GLuint getId() const {
+        if (gl_texture_ == 0) {
```

Comment:
We generally don't check in code for debugging a particular issue: do you think this is going to be useful in other contexts?